AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -130,6 +130,7 @@ private var runtimeTargets: [Target] {
                 .SmithyIdentity,
                 .SmithyRetriesAPI,
                 .SmithyRetries,
+                .SmithyTimestamps,
```

Comment:
The new AWS-specific clock skew provider uses `SmithyTimestamps` to read the date header in HTTP responses, hence the dependency is added.